css-ui-4/Overview.bs

```diff
@@ -2135,6 +2135,14 @@ Switching appearance: the 'appearance' property</h3>
 	giving them a <dfn export>primitive appearance</dfn>
 	where CSS can be used to restyle them.
 
+	Using ''appearance: base'' gives <a>widgets</a> a <dfn export>base
+	appearance</dfn>. <a>Widgets</a> in <span>base appearance</span> have their
+	<a>native appearance</a> suppressed where CSS can be used to restyle them,
+	just like <a>primitive appearance</a>, but also have additional styling
+	capabilities and default styles which are usable and interoperable. This is
```

Comment:
I would remove "additional styling capabilities", in general we should try to make as many capabilities available to `none` as well as `base`. 

`auto` is more the odd one out, where the UA can choose to not honor (parts or all) custom styling from authors, because the platform doesn't support it.

Reply:
Thanks for taking a look! done